trellis/local-development-setup.md

```diff
@@ -13,7 +13,7 @@ Development is handled by [Vagrant](https://www.vagrantup.com/) in Trellis. Our
 
 1. Configure your site(s) based on the [WordPress Sites docs](https://roots.io/trellis/docs/wordpress-sites/) and read the [development specific](https://roots.io/trellis/docs/wordpress-sites/#development) ones.
 2. Make sure you've edited both `group_vars/development/wordpress_sites.yml` and `group_vars/development/vault.yml`.
-3. Optionally configure the IP address at the top of the `Vagrantfile` to allow for multiple boxes to be run concurrently (default is `192.168.50.5`).
+3. Optionally configure the IP address `vagrant_ip:` at the top of `vagrant.default.yml` to allow for multiple boxes to be run concurrently (default is `192.168.50.5`).
```

Comment:
Minor thing but I'd rather just say:

> Optionally configure the IP address via `vagrant_ip` in `vagrant.default.yml` to allow for multiple boxes to be run concurrently (default is `192.168.50.5`).

Reply:
Looking at the Vagrant file (https://github.com/roots/trellis/blob/master/Vagrantfile#L12) it would appear that there is logic to read from a vagrant.local.yml file. Wouldn't it make more sense to have users create that file and add vagrant_ip there to override the default values?

Reply:
Probably 👍